Changelog.md

```diff
@@ -9,6 +9,12 @@ every new version is a new major version.
 
 ## 19 - 2023-03-13
 
+### Fix
```

Comment:
Would be nice to have main merged into current branch and updated accordingly (changes will now relate to v21)